jekyll/_cci2/url-orbs.adoc

```diff
@@ -0,0 +1,173 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4+
+---
+= URL orbs
+:page-description: How to manage URL orbs allow-lists
+:experimental:
+:icons: font
+:page-layout: classic-docs
+
+[#introduction]
+== Introduction
+
+The ability for an organization to use private orbs is no longer dependent on publishing orbs. URL-referenced orbs offer a way to create orbs as well as control access within the organization.
+
+The use of URL orbs is recommended for private orbs where there are additional trust and access controls within the VCS.
```

Comment:
~\~\~ We don't necessarily need to call it out, but URL orbs can be hosted anywhere, not just in a VCS.

Because we have special integrations with the VCS providers we can access restricted URLs on behalf of customers, but there's no reason we couldn't also build something that allows hosting orbs on S3 etc.